include/motor-control/core/motor_messages.hpp

```diff
@@ -34,8 +34,9 @@ struct Move {        // NOLINT(cppcoreguidelines-pro-type-member-init)
 const uint8_t NO_GROUP = 0xff;
 
 enum class AckMessageId : uint8_t {
-    complete_without_condition = 0x1,
-    stopped_by_condition = 0x2
+    complete = 0x1,
```

Comment:
I'm a little confused by this - I think `complete_without_condition` and `stopped_by_condition` are perfectly adequate for this, right? `complete_without_condition` is the default completion for moves without condition, but in the context of home it probably means there was an error; `stopped_by_condition` is a good value for when a condition, well, stopped the move.